uk.co.compucorp.civicrm.civihrsampledata/civihrsampledata.php

```diff
@@ -0,0 +1,116 @@
+<?php
```

Comment:
Yeah, we discussed about the uk.co.compucorp.civicrm prefix, but I don't remember anything about the extension name. 
I know there are extensions not following any pattern, but, as I mentioned in a comment on https://github.com/civicrm/civihr/pull/1203, most of our extensions names start with hr (including L&A which is uk.co.compucorp.civicrm.hrleaveandabsences). Even the core extension which just got merged follow this (uk.co.compucorp.civicrm.hrcore). So, unless we decide to use one specific convention, I believe the best thing to do is to follow what we've been doing for most of the extensions.